lawnchair/res/values/strings.xml

```diff
@@ -343,6 +343,56 @@
     <string name="smartspace_widget">At a Glance</string>
     <string name="smartspace_widget_description">What to show</string>
 
+    <!-- Personality provider -->
+    <string name="personality_provider">Contextual messages</string>
+    <string-array name="smartspace_personality_greetings_morning">
+        <item>Good morning!</item>
+        <item>Have a great day!</item>
+        <item>It’s a beautiful day outside!</item>
+        <item>Rise and shine!</item>
+        <item>Top of the morning to you!</item>
+        <item>Ready to start the day?</item>
+    </string-array>
+    <string-array name="smartspace_personality_greetings_morning_subtitles">
+        <item>Wishing you a pleasant start to your day.</item>
+        <item>May your day be filled with joy and success.</item>
+        <item>Don\'t forget to enjoy the sunshine and fresh air.</item>
+        <item>Time to wake up and make things happen!</item>
+        <item>A cheerful greeting for a cheerful morning.</item>
+        <item>Let\'s make today amazing!</item>
+    </string-array>
+    <string-array name="smartspace_personality_greetings_evening">
+        <item>What’s up?</item>
+        <item>Good evening!</item>
+        <item>How did your day go?</item>
+        <item>How was your day?</item>
+        <item>Evening! How are you?</item>
+        <item>Hope you had a good day!</item>
+    </string-array>
+    <string-array name="smartspace_personality_greetings_evening_subtitles">
+        <item>Just checking in to see how your evening is going.</item>
+        <item>Curious to hear about your day\'s adventures.</item>
+        <item>Hoping your day was productive and fulfilling.</item>
+        <item>A friendly greeting for a wonderful evening.</item>
+        <item>Time to unwind and enjoy the rest of your day.</item>
+    </string-array>
+    <string-array name="smartspace_personality_greetings_night">
+        <item>Sleep well!</item>
+        <item>See you tomorrow!</item>
+        <item>Good night!</item>
+        <item>Don’t stay up too late!</item>
+        <item>Night night!</item>
+        <item>Sweet dreams!</item>
+    </string-array>
+    <string-array name="smartspace_personality_greetings_night_subtitles">
+        <item>May you have a restful sleep.</item>
+        <item>Looking forward to seeing you again tomorrow.</item>
+        <item>Wishing you a peaceful and pleasant night.</item>
+        <item>Remember to sleep for your health.</item>
+        <item>A cozy goodnight message for a sweet slumber.</item>
+        <item>May your dreams be filled with wonder and joy.</item>
+    </string-array>
```

Comment:
Please help this is so robotic

Reply:
I haven't used old lawnchair, but this just shows greetings, right?
In my app I use ones from crdroid: https://raw.githubusercontent.com/crdroidandroid/android_packages_apps_Launcher3/14.0/res/values/cr_strings.xml (those starting with `quickspace_grt_`). I think bananadroid had some nice ones too.

Reply:
you can take greetings from the following 
https://github.com/BananaDroid-Revived/android_packages_apps_Launcher3/blob/14/res/values/banana_strings.xml
https://github.com/BootleggersROM/packages_apps_Launcher3/blob/tirimbino/res/values/bootleg_strings.xml
https://github.com/ProjectMatrixx/android_packages_apps_Launcher3/blob/14.0/res/values/cr_strings.xml

Reply:
Thanks, will look into it.

Reply:
Maybe a bad idea, but how about a small ML model to provide greetings?

Reply:
> Maybe a bad idea, but how about a small ML model to provide greetings?

Hi @suphon-t, I'm not sure how this would work. I would love some examples here, should the model show greetings to the user when it thinks it is the right time to do so or something else?

Thanks.